daemon/logger/journald/journald.go

```diff
@@ -48,10 +49,17 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		name = name[1:]
 	}
 
+	// parse log tag
+	tag, err := loggerutils.ParseLogTag(ctx, "")
+	if err != nil {
+		return nil, err
+	}
+
 	vars := map[string]string{
 		"CONTAINER_ID":      ctx.ContainerID[:12],
 		"CONTAINER_ID_FULL": ctx.ContainerID,
 		"CONTAINER_NAME":    name,
+		"CONTAINER_TAG":     tag,
```

Comment:
I want to use `CONTAINER_TAG` to centralize log collection for services. If docker container does not provide tag, I probably don't want to centralize logs from this service. Using `CONTAINER_TAG` allows me to filter by it, while `SYSLOG_IDENTIFIER` probably does not.

It is also more consistent with existing parameters.